### ISSUE TITLE
Importing / Recording - Icon search is finicky. (Try to find the right keyword for the hot air balloon or any of the scales!)

### ISSUE THREAD

**Issue by alexchippendale** [kidtalkdev/KidTalkRemainingTasks#12]
It feels like usually no icons are suggested. For instance, nothing comes up for food, fruit, drink, etc. I'm not sure this is a lack of icons. Maybe just a lack of them showing up?

**Comment by alexchippendale** [comment]
We may be able to pull from a larger pool of icons, and will be working on improving icon selection based on text search.

**Comment by jkhartshorne** [comment]
@alexchippendale - any progress on this?

**Comment by alexchippendale** [comment]
Hi @jkhartshorne we have a dev working on it right now. I'll let you know when we deploy any enhancements to the icon searcher.

**Comment by jkhartshorne** [comment]
@alexchippendale - thanks. To be clear, it's not just about the number of icons. It's that you can't get many/most of them to show up through text search.

**Comment by alexchippendale** [comment]
@jkhartshorne Makes sense, we're working on both improving the text search and expanding the icon set.

**Comment by alexchippendale** [comment]
The icon searcher has now been updated on staging to be better at displaying icons. 

We will continue to refine it whenever possible and will also be adding more icons.

**Comment by alexchippendale** [comment]
This has been deployed to production as well. I'll keep it open for now since we'll also be adding more icons and may refine more.

**Comment by jkhartshorne** [comment]
Have you added the new icons or refined search? If there is a spreadsheet that you are adding keywords to, maybe I could have a student help? There's a lot of obvious keywords missing. For instance, "food" and "drink" don't bring up any foods or drinks!

**Comment by alexchippendale** [comment]
The search has been modified to include more icons and relevant icons. I will be discussing the status of the additional icons and further refinement later today with the team.
